G/GMP/build_tarballs.jl

```diff
@@ -4,27 +4,20 @@ using BinaryBuilder
 
 # Collection of sources required to build GMP
 name = "GMP"
-version = v"6.1.2"
+version = v"6.2.0"
 
 sources = [
     ArchiveSource("https://gmplib.org/download/gmp/gmp-$(version).tar.bz2",
-                  "5275bb04f4863a13516b2f39392ac5e272f5e1bb8057b18aec1c9b79d73d8fb2"),
+                  "f51c99cb114deb21a60075ffb494c1a210eb9d7cb729ed042ddb7de9534451ea"),
     DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/gmp-*
 
-# Update config.status
-update_configure_scripts
```

Comment:
Despite the comment, I think this actually updates `config.guess` and `config.sub` ; both are fairly recent in GMP 6.2.0, so this shouldn't be necessary anymore